accelerated-table-creation.md

```diff
@@ -12,13 +12,9 @@ TiDB v7.6.0 引入了系统变量 [`tidb_ddl_version`](https://docs.pingcap.com/
 
 合并后的批量建表语句在同一个事务内执行，如果其中一个语句失败，所有语句都会失败。
 
-> **警告：**
->
-> TiDB 加速建表目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
-
 ## 与 TiDB 工具的兼容性
 
-- [TiCDC](/ticdc/ticdc-overview.md) 暂不支持同步通过 TiDB 加速创建的表。
+- TiDB v8.3.0 版本之前 [TiCDC](/ticdc/ticdc-overview.md) 暂不支持同步通过 TiDB 加速创建的表。TiDB v8.3.0 版本之后可以正常同步。
```

Comment:
```suggestion
- 在 TiDB v8.3.0 之前的版本中，[TiCDC](/ticdc/ticdc-overview.md) 不支持同步通过 TiDB 加速创建的表。从 v8.3.0 开始，TiCDC 可以正常同步这类表。
```

If this compatibility issue has been resolved in v8.3.0, please submit two separate PRs:

- For the PR targeting `release-8.3` branch: update the compatibility description to:

```
## 与 TiDB 工具的兼容性

- 在 TiDB v8.3.0 之前的版本中，[TiCDC](/ticdc/ticdc-overview.md) 不支持同步通过 TiDB 加速创建的表。从 v8.3.0 开始，TiCDC 可以正常同步这类表。
```

- For the PR targeting `master` branch (to be cherry-picked to v8.4): remove the TiCDC compatibility section and provide a release note for this change in `release-8.3.0.md`.

```diff
- ## 与 TiDB 工具的兼容性

- [TiCDC](/ticdc/ticdc-overview.md) 暂不支持同步通过 TiDB 加速创建的表。
```